CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ endif()
 
 function(boost_url_setup_properties target)
     target_compile_features(${target} PUBLIC cxx_constexpr)
+    target_compile_features(${target} PUBLIC cxx_std_11)
```

Comment:
Why are you fiddling with the CML?

Reply:
The problem is because this is wrong or because it's unrelated to the PR?

Reply:
I have no idea, I am not a cmake expert, so I don't know what impact this has. And the other libraries don't have this line. So why don't they? The people who worked on the CMakeLists.txt decided not to include it. Why? Maybe it is not available in older cmakes. Maybe it causes a problem with the Boost superproject. Maybe they just didn't know it exists. In any event, this change is unrelated to improving documentation and it needs to be in its own pull request, reviewed by people who know Cmake better than I do (and who know the impact on the Boost superproject).

Reply:
Assuming it's because it's unrelated to the PR, this is because this draft was moved from the fork because the PR in the fork didn't get the preview. The intention of the draft was to share the state of the documentation and so you could have a look at what I have locally to give me instructions. If anything is useful, I would just cherry-pick it into another PR. In any case, this doesn't seem to be working. I'll just close this PR then.

Reply:
> this change is unrelated to improving documentation 

Since (i) this is a sufficient condition to remove the `cxx_std_11` requirement from this target and (ii) I'm closing this PR anyway, I won't argue in favor of it.

Reply:
the c++11 change should be opened as its own separate pull request